src/Hosting/Hosting/src/Startup/IRejectStartup.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.AspNetCore.Hosting
+{
+    /// <summary>
+    /// A marker interface used to indicate the <see cref="IHostBuilder"/> or <see cref="IWebHostBuilder"/>
+    /// does not support any of the following extension methods:
+    /// * <see cref="WebHostBuilderExtensions.Configure(IWebHostBuilder, Action{AspNetCore.Builder.IApplicationBuilder})"/>
```

Comment:
Need to use <list> for this: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#list